# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../portmap/no-pathto-libwrap.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

For lib64 and cross builds, ...

  - Rene Rebe <rene@exactcode.de>

--- portmap_4/Makefile.vanilla	2009-04-09 16:07:22.000000000 +0200
+++ portmap_4/Makefile	2009-04-09 16:08:37.000000000 +0200
@@ -75,8 +75,8 @@
 
 all:	portmap pmap_dump pmap_set
 
-portmap: $(OBJECTS) $(WRAP_DIR)/libwrap.a
-	$(CC) $(CFLAGS) -o $@ $(OBJECTS) $(WRAP_DIR)/libwrap.a $(LIBS)
+portmap: $(OBJECTS)
+	$(CC) $(CFLAGS) -o $@ $(OBJECTS) -lwrap $(LIBS)
 
 pmap_dump: pmap_dump.c
 	$(CC) $(CFLAGS) -o $@ $? $(LIBS)
